cmake/FindSenseGlove.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Istituto Italiano di Tecnologia (IIT)
+# @author Kourosh Darvish <kourosh.darvish@iit.it>
+
+# Finds the Sense Glove SDK
+#
+# This will define the following variables::
+#
+#   SenseGlove_FOUND         - True if the system has the Sense Glove SDK
+#   SenseGlove_INCLUDE_DIR  - The include directory of the SDK
+#   SenseGlove_LIBRARIES     - The name of the libraries to link against.
+###############################
+
+# Check Directory of the SenseGlove_DIR
+if(NOT DEFINED ENV{SenseGlove_DIR})
+  message( FATAL_ERROR "Environment variable {SenseGlove_DIR} is not defined: $ENV{SenseGlove_DIR}" )
```

Comment:
what would be printed in the end of the error if `$ENV{SenseGlove_DIR}` is not defined?

Reply:
probably an empty string! I do not remember! I can remove it!

Reply:
Ok, also below

Reply:
I removed it.